tests/test_config_image.py

```diff
@@ -2455,6 +2460,29 @@ def test_template():
     print(cl.output)
     assert "Removing item 0 from image.random_seed." in cl.output
 
+    # Read a template config from a file
+    config8 = galsim.config.ReadConfig('config_input/template.yaml')[0]
+    galsim.config.ProcessAllTemplates(config8)
+    for key in config3:
+        print(key)
+        print(config4[key])
+        print(config8[key])
+        assert config8[key] == config4[key]
+    assert config8 == config4
+
+    # Make sure nested templating works
```

Comment:
I think the section above that tests the config_input/template.yaml file also tests nesting.. might want to note it above also.

Reply:
No.  The previous test just confirms that template.yaml is equivalent to the thing we already tested above in python code.  config9 is the one that tests your use case of nesting templates.